library/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -152,6 +154,7 @@ protected void onDetachedFromWindow() {
             mDisplayOrientationDetector.disable();
         }
         super.onDetachedFromWindow();
+        mCallbacks.cleanup();
```

Comment:
when CameraView has become detached from window, call this simple cleanup method of mCallbacks